src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -225,6 +225,10 @@ public long LastRaftCommitIndex
                                     {
                                         if (notification == null)
                                             break;
+
+                                        if (notification.TryGet("Id",out string id) && id != nameof(ClusterTopologyChanged))
```

Comment:
What is this about?

Reply:
It seems that even if the notification is of a different type (but same base type) it still will serialize it in a bad manner.